Modern/wrapperPlugins/src/test/java/org/bonej/wrapperPlugins/ElementFractionWrapperTest.java

```diff
@@ -65,43 +61,15 @@ public void testNonBinaryImageCancelsElementFraction() throws Exception {
 	}
 
 	@Test
-	public void testNoCalibrationShowsWarning() throws Exception {
-		CommonWrapperTests.testNoCalibrationShowsWarning(IMAGE_J,
+	public void testNullImageCancelsElementFraction() throws Exception {
+		CommonWrapperTests.testNullImageCancelsPlugin(IMAGE_J,
 			ElementFractionWrapper.class);
 	}
 
-	@Test
-	public void testWeirdSpatialImageCancelsPlugin() throws Exception {
-		// Mock UI
-		final UserInterface mockUI = mock(UserInterface.class);
-		final SwingDialogPrompt mockPrompt = mock(SwingDialogPrompt.class);
-		when(mockUI.dialogPrompt(anyString(), anyString(), any(), any()))
-			.thenReturn(mockPrompt);
-		IMAGE_J.ui().setDefaultUI(mockUI);
-
-		// Create an hyperstack with no calibration
-		final DefaultLinearAxis xAxis = new DefaultLinearAxis(Axes.X);
-		final DefaultLinearAxis cAxis = new DefaultLinearAxis(Axes.CHANNEL);
-		final Img<DoubleType> img = ArrayImgs.doubles(5, 5);
-		final ImgPlus<DoubleType> imgPlus = new ImgPlus<>(img, "Test image", xAxis,
-			cAxis);
-
-		// Run command
-		final CommandModule module = IMAGE_J.command().run(
-			ElementFractionWrapper.class, true, "inputImage", imgPlus).get();
-
-		assertTrue("A non 2D and 3D image should have cancelled the plugin", module
-			.isCanceled());
-		assertEquals("Cancel reason is incorrect", CommonMessages.WEIRD_SPATIAL,
-			module.getCancelReason());
-		verify(mockUI, timeout(1000)).dialogPrompt(anyString(), anyString(), any(),
-			any());
-	}
-
+	// TODO Rewrite with a hyper stack
```

Comment:
As discussed, a hyper-stack should probably report BV/TV for each channel at each time point.

Reply:
Cheers for the review